cmake/modules/FindSccache.cmake

```diff
@@ -0,0 +1,26 @@
+find_program(CCACHE_PROGRAM sccache)
```

Comment:
This file looks very similar to sccache.cmake. Why are both necessary?

Reply:
Oops, good catch. You're right, this was a copy/paste error. Removed.